themes/classic/modules/ps_shoppingcart/modal.tpl

```diff
@@ -56,12 +56,20 @@
               {else}
                 <p class="cart-products-count">{l s='There is %product_count% item in your cart.' sprintf=['%product_count%' =>$cart.products_count] d='Shop.Theme.Checkout'}</p>
               {/if}
-              <p><strong>{l s='Total products:' d='Shop.Theme.Checkout'}</strong>&nbsp;{$cart.subtotals.products.value}</p>
-              <p><strong>{l s='Total shipping:' d='Shop.Theme.Checkout'}</strong>&nbsp;{$cart.subtotals.shipping.value} {hook h='displayCheckoutSubtotalDetails' subtotal=$cart.subtotals.shipping}</p>
+              <p><span><strong>{l s='Total products:' d='Shop.Theme.Checkout'}</strong></span>&nbsp;<span><strong>{$cart.subtotals.products.value}</strong></span></p>
+              <p><span>{l s='Total shipping:' d='Shop.Theme.Checkout'}</span>&nbsp;<span><strong>{$cart.subtotals.shipping.value} {hook h='displayCheckoutSubtotalDetails' subtotal=$cart.subtotals.shipping}</strong></span></p>
+
+              {if $cart.totals.total_including_tax.value != $cart.totals.total_excluding_tax.value && $cart.totals.total.value == $cart.totals.total_excluding_tax.value}
+                <p><span>{$cart.totals.total.label}&nbsp;{$cart.labels.tax_short}</span>&nbsp;<span>{$cart.totals.total.value}</span></p>
+                <p class="product-total"><span><strong> {$cart.totals.total_including_tax.label}</strong></span>&nbsp;<span><strong>{$cart.totals.total_including_tax.value}</strong></span></p>
+              {else}
+                <p class="product-total"><span><strong>{$cart.totals.total.label}&nbsp;{$cart.labels.tax_short}</strong></span>&nbsp;<span><strong>{$cart.totals.total.value}</strong></span></p>
+              {/if}
+
               {if $cart.subtotals.tax}
-              	<p><strong>{$cart.subtotals.tax.label}</strong>&nbsp;{$cart.subtotals.tax.value}</p>
+                <p class="no-spacing">{$cart.subtotals.tax.label}&nbsp;<strong>{$cart.subtotals.tax.value}</strong></p>
```

Comment:
`<p class="no-spacing">`

I'm not a fan of that class name. How about using something semantic like I suggested above?

Reply:
I can change it to "product-tax"